code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -69,12 +69,12 @@
 		if(dna.mutantrace == "slime")
 			tally *= 2
 		else
-			tally -= 10
+			tally -= 5
 
 	if(reagents.has_reagent("frostoil") && dna.mutantrace == "slime")
 		tally *= 5
 
-	return max((tally+config.human_delay), -1) //cap at -1 as the 'fastest'
+	return max((tally+config.human_delay), -100) //cap at -100 as the 'fastest'
```

Comment:
Neither will this because movement speed is maxed in client/move() at 1 because of division by 0 runtimes (thanks to all these speedups/slowdowns being additive instead of multiplicative.

It's not a well designed system.